### PR TITLE
Cheatsheet/georgian language

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New (Georgian) Language Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

Georgian (Kartuli) is the official language spoken in Georgia.
The cheatsheet with basic words and phrases for the same have been
created and should be included on IA goodies.

> Source: Friend who is a native speaker (linguistic student)

Provide an overview of the changes this pull request introduces.
Georgian language cheatsheet JSON added
###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?
###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/georgian_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@NaveenKarippai Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png)

**Comment by NaveenKarippai** [comment]
@zekiel @moollaza it would be highly appreciated if there is an update on PR review status.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png?P7phSG4YFk)

File: share/goodie/cheat_sheets/json/language/georgian.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
+{
+    "id": "georgian_cheat_sheet",
+    "name": "Georgian Cheat Sheet",
+    "description": "Basic Georgian words and phrases",
+
+    "metadata": {
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/language/georgian.json]: Please add a source.

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @gaulrobe The cheatsheet was written by a friend of mine (who is a native speaker). There is no source to attribute here and AFAIK it's optional. Thanks!

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @gaulrobe done!

**Comment by zekiel** [comment]
@NaveenKarippai  we'll get this out to the community for a language review! also re-installing the latest updates on beta bc it looks like the last deploy was abandoned.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png?xR4owJyR4Y)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png?sryP4Gxyu3)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png?6bHZCek0Ci)

**Comment by tagawa** [comment]
Have sent a request for language reviews on Twitter: https://twitter.com/duckduckgo/status/760474282388516864

**Comment by tagawa** [comment]
@NaveenKarippai We haven't had any language feedback so if the source you used is reliable, then this should be OK. Please could add the source name and URL to the cheat sheet (after the description) like this?

    "metadata":{
        "sourceName": "Wikipedia",
        "sourceUrl" : "https://en.wikipedia.org/wiki/Harry_Potter"
    },

**Comment by NaveenKarippai** [comment]
@tagawa The cheatsheet had been done by a friend of mine who indeed is a native speaker. I'd be unhappy to include a "meta" source info as there aren't any. Could you please give a quick feedback if that works. Thanks!

**Comment by tagawa** [comment]
@NaveenKarippai I see. We could use any website that contains the same basic phrases. I'll see what I can find but if you know of any, feel free to add them.

**Comment by tagawa** [comment]
@NaveenKarippai I think I've found a page we can use as a source: http://www.omniglot.com/language/phrases/georgian.php
It seems to have similar phrases, and we've also used this website as a source for other language cheat sheets. Please could you add this as the source name and URL? Here's an example: https://github.com/duckduckgo/zeroclickinfo-goodies/blob/41f3b505513fc4f21a148c7b86facb017d1fcd1a/share/goodie/cheat_sheets/json/language/swahili.json#L5

**Comment by tagawa** [comment]
Hi @NaveenKarippai. Please could you add this code below the `description` property?

    "metadata": {  
        "sourceName": "Omniglot",
        "sourceUrl": "http://www.omniglot.com/language/phrases/georgian.php"
    },

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**